backend/entry.ts

```diff
@@ -4,10 +4,10 @@ import {Angular2Adapter} from './adapters/angular2';
 let channel = {
   sendMessage: (message) => {
     console.log('Inspected script sending: ', message);
-    return window.postMessage({
+    return window.postMessage(JSON.parse(JSON.stringify({
```

Comment:
@vanessayuenn could you provide some rationale behind this? I just want to understand the reasons behind this.